light-prover-sdk/src/generics.ts

```diff
@@ -0,0 +1,133 @@
+import { 
+    IDL_VERIFIER_PROGRAM_ZERO,
+    IDL_VERIFIER_PROGRAM_ONE,
+    IDL_VERIFIER_PROGRAM_TWO,
+    IDL_VERIFIER_PROGRAM_STORAGE,
+    IDL_MERKLE_TREE_PROGRAM
+} from "../../light-zk.js/src/idls";
+
+export type VerifierIdls = 
+    typeof IDL_VERIFIER_PROGRAM_ZERO |
+    typeof IDL_VERIFIER_PROGRAM_ONE |
+    typeof IDL_VERIFIER_PROGRAM_TWO |
+    typeof IDL_VERIFIER_PROGRAM_STORAGE |
+    typeof IDL_MERKLE_TREE_PROGRAM;
+
+// type IDL0 = typeof IDL_VERIFIER_PROGRAM_ZERO;
+// type IDL2 = typeof IDL_VERIFIER_PROGRAM_TWO;
```

Comment:
pls remove unnecessary comments

Reply:
I will keep your feedback in mind, for a future refactoring of prover generics.